src/Workload/Microsoft.Maui.Sdk/Sdk/AutoImport.in.props

```
@@ -7,7 +8,7 @@
     </MauiXaml>
   </ItemDefinitionGroup>
 
-  <ItemGroup Condition=" '$(UseMaui)' == 'true' and ('$(ImplicitUsings)' == 'true' or '$(ImplicitUsings)' == 'enable') ">
+  <ItemGroup Condition=" '$(UseMaui)' == 'true' and ('$(ImplicitUsings)' == 'true' or '$(ImplicitUsings)' == 'enable') and '$(TargetPlatformIdentifier)' != '' and $([MSBuild]::VersionEquals($(TargetFrameworkVersion), '@MAUI_DOTNET_VERSION@')) ">
```

I think `$(TargetPlatformIdentifier)` is for some reason `windows` in a `net6.0` project. What is the issue you're hitting?

I don't think this would have a way to add the same `@(Using)` twice? Even so, it would probably still work.

---

Does every line need to check if `$(TargetFrameworkVersion)` is blank?

I should test a `net6.0-android31;net6.0-android32` class library.

---

Android works:
```
<Project Sdk="Microsoft.NET.Sdk">
	<PropertyGroup>
		<TargetFrameworks>net7.0;net7.0-android32;net7.0-android33</TargetFrameworks>
		<EnablePreviewFeatures>true</EnablePreviewFeatures>
	</PropertyGroup>
</Project>
```
It's because we have the `TargetPlatformIdentifier` matching `android` checks. But I think MAUI should just check if `$(TargetFrameworkVersion)` is blank:

```suggestion
  <ItemGroup Condition=" '$(UseMaui)' == 'true' and ('$(ImplicitUsings)' == 'true' or '$(ImplicitUsings)' == 'enable') and '$(TargetFrameworkVersion)' != '' and $([MSBuild]::VersionEquals($(TargetFrameworkVersion), '@MAUI_DOTNET_VERSION@')) ">
```